vue-frontend/src/components/contact/ThankyouScheduleMeeting.vue

```diff
@@ -0,0 +1,178 @@
+<template>
+  <section class="tw-relative">
```

Comment:
Again top space is very little. Check it with other sections.
Where is background color?